### ISSUE TITLE
Datepicker popup position broken when upgrading to 3.0.0-alpha.7

### ISSUE THREAD

**Issue by lafriks** [nathanreyes/v-calendar#1058]
Position is correct in 3.0.0-alpha.6

![attels](https://user-images.githubusercontent.com/165205/152303329-db9d5de8-dcfe-43d2-8363-74af642be428.png)

**Comment by stevebauman** [comment]
I'm able to reproduce this. Default layout is entirely gone in Alpha 7. Downgrading to Alpha 6 returns everything back to normal 👍

**Comment by stevebauman** [comment]
Sorry my mistake -- I re-read the readme and I saw the upgrade notes. Everything works fine!

@lafriks Did you read the following update?

> :warning: **As of `v3.0.0-alpha.7`, all installation methods require manual import of component styles. This is due to Vite >build restrictions in libary mode.**
>
>```js
>import 'v-calendar/dist/style.css';
>```

**Comment by lafriks** [comment]
Yes I did css import, all styles are applied as seen in screenshot, it just that position is off

**Comment by nathanreyes** [comment]
I'll take a look

**Comment by nathanreyes** [comment]
@lafriks Having trouble duping this. Can you provide your template html? I'm curious how your date picker slot is configured.

**Comment by mbrodala** [comment]
@nathanreyes Can you add this to the Github release notes for alpha7?

https://github.com/nathanreyes/v-calendar/tree/next#use-plugin

**Comment by lafriks** [comment]
After deeper inspection it looks like it has to do something with Carbon Design System css, it somehow conflicts with v-calendar css for CDS modals but I can't seem to find exact reason for that

**Comment by lafriks** [comment]
Ok, it looks like it was because of moving popperjs as direct instead of peer dependency as it was before. Problem is because of this issue floating-ui/floating-ui#1488. Updating `@popperjs/core` to `2.11.4` fixes the problem
